<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="3" hostname="LAPTOP-OEIBQQ4S" failures="1" tests="2" name="Test(failed)" time="0.098" errors="0" timestamp="2022-06-27T15:15:56 IST">
  <testcase classname="TestNGStudy.FailTCExecution" name="TC8" time="0.018">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at TestNGStudy.FailTCExecution.TC8(FailTCExecution.java:29)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- TC8 -->
  <testcase classname="TestNGStudy.FailTCExecution" name="TC10" time="0.005"/>
  <testcase classname="TestNGStudy.FailTCExecution" name="TC6" time="0.0">
    <ignored/>
  </testcase> <!-- TC6 -->
  <testcase classname="TestNGStudy.FailTCExecution" name="TC7" time="0.0">
    <ignored/>
  </testcase> <!-- TC7 -->
  <testcase classname="TestNGStudy.FailTCExecution" name="TC9" time="0.0">
    <ignored/>
  </testcase> <!-- TC9 -->
</testsuite> <!-- Test(failed) -->
